geom/type_envelope.go

```diff
@@ -157,25 +178,7 @@ func (e Envelope) Area() float64 {
 	return (e.max.X - e.min.X) * (e.max.Y - e.min.Y)
 }
 
-// ExpandBy calculates a new version of this envelope that is expanded in the x
```

Comment:
This wasn't used by any (known) users of the library. And this method is a bit out of harmony with the other sorts of envelope methods. With the NaN/Inf validation changes, it was going to need to change to have error handling as well, which made the signature a bit awkward (`Envelope, bool, error`). I thought it was best to just remove it.